cmake/SearchForStuff.cmake

```diff
@@ -583,13 +600,20 @@ if (PKG_CONFIG_FOUND)
   # First and preferred option is to look for bullet standard pkgconfig,
   # so check it first. if it is not present, check for the OSRF
   # custom bullet2.82.pc file
-  pkg_check_modules(BULLET bullet>=2.82)
+  find_package(BULLET CONFIG 2.82)
```

Comment:
For backward compatibility, can we search via `find_package`  only if `    pkg_check_modules` fails?

Reply:
I will try.